lesson05/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,752 @@
+}
+
+// Background image by Matt Hoffman on Unsplash
+// https://unsplash.com/photos/ZUUsGnG5zwc
+.header {
+  grid-area: header;
+  @include flex-row;
+  justify-content: space-between;
+  background: 
+    linear-gradient(rgba(0, 0, 0, .4), rgba(0, 0, 0, .4)),
+    url('../img/matt-hoffman-ZUUsGnG5zwc-unsplash-height-1000.jpg') no-repeat left fixed;
+  background-color: $tentondarker;
+  height: 80vh;
+  &_name-logo {
+    width: auto;
+    padding: 1% 0 0 5%;
+    color: $white;
+    text-transform: uppercase;
+    font-family: $nav-font;
+    font-size: 7rem;
+    @include flex-column;
+    align-items: flex-start;
+    justify-content: center;
+    & p:nth-of-type(1) {
+      margin: 0;
+    }
+    & p:nth-of-type(2) {
+      font-size: 5rem;
+      align-self: center;
+      margin-top: 0;
+    }
+  }
+  &_nav {  // Nav styling
+    font-family: $nav-font;
+    text-transform: uppercase;
+    font-size: 2rem;
+    color: $white;
+    padding: 1% 5% 0 0;
+    &_items {
+      @include flex-row;
+      & li:not(:last-child) {
+        padding-right: 4rem;
+      }
+      & a {
+        @include underline-transition;
+      }
+    }
+  }
+}
+
+.main-container {
+  grid-area: main;
+  padding: 2% 4% 6% 5%;
+  @include flex-column;
+  align-items: center;
+  &_intro {  // Intro text styling
+    font-size: 1.5rem;
+    padding: 2% 20% 2% 20%;
+    line-height: 1.8;
+    letter-spacing: .025em;
+    text-align: center;
+  }
+  & h2 {
+    text-align: center;
+  }
+  &_articles {  // Two main image/text section styling
+    @include flex-row;
+    justify-content: center;
+    padding: 2% 0 2% 0;
+    margin-bottom: 2%;
+    &_wrapper {
+      width: 100%;
+      height: auto;
+      overflow: hidden;
+      background: $black;
+      & img { 
+        width: 100%;
+        height: auto;
+        opacity: .7; 
+        transition: transform 10s ease-out .15s, opacity .5s ease-out; // Image zoom transition
+      }
+      &:hover img {
+        opacity: 1;  // Image fade in/out transition
+        transform: scale(1.5);
+      }
+    }
+    & article {
+      width: 100%;
+      @include flex-column;
+      align-items: center;
+      margin: 0 1.25rem 0 1.25rem;
+    }
+    &_text {  // Overlay text on two middle images
+      text-align: center;
+      width: 90%;
+      height: 30%;
+      background-color: $white;
+      border-bottom: 1px solid $black;
+      position: relative;
+      margin-top: -90px;
+      padding: 4% 4% 4% 4%;
+      letter-spacing: .025em;
+      line-height: 1.4;
+      & h4 {
+        margin-top: 0;
+      }
+    }
+  }
+  &_instagram { // To be API feed
+    display: grid;
+    grid-template-columns: repeat(5, 250px);
+    grid-template-rows: 1fr;
+    gap: 1rem;
+    &_wrapper {
+      overflow: hidden;
+      background: $black;
+      & img {
+        opacity: .7;
+        transition: opacity .5s ease-out;
+      }
+      &:hover img {
+        opacity: 1;
+      }
+    }
+  }
+}
+
+// Footer styling
+.footer {
```

Comment:
As mentioned below, we can just use the footer element as the selector here, no need for a redundant class name.